community/glossary.md

```diff
@@ -0,0 +1,195 @@
+
+Another way of raising funds for grain-supported projects will be licensing along the lines of a Harberger Grain License 3, whereby proprietary dependencies of open projects will need to pay a tax, and the tax consists of buying and burning a project’s grain.
+
+#### SourceGrain as Economic Scaffolding
+
+Grain can serve a lot of economic functions for open-source projects:
+
+ - It provides a new way to support an open-source project: by buying its grain.
+ - It provides a way for contributors to earn money for their work: selling the grain they earn for contributing to the project.
+ - It provides funding for widely-depended on open-source infrastructure: such projects will earn grain in all of their many dependencies.
+ - It lets users of a project who need prioritized support or access to acquire it, by paying the developers.
+ - It provides rewards for early investors in successful projects, both via grain appreciation, and via dividends from dependent projects.
+
+And in this light, you can think of grain as as producing general-purpose economic infrastructure for open-source projects. It would not be inaccurate to say that SourceCred exists so that grain can be a reality.
+
+#### In the wild
+
+[Here](https://github.com/sourcecred/sourcecred/issues/1425) is an example of an issue 
+discussing grain in context of the sourcecred software, and [here](https://observablehq.com/@decentralion/credsperiment-week-16) is an observablehq notebook that shows how
+grain works in the current "cred-speriment."
+
+#### References
+
+ - [Grain on sourcecred discourse](https://discourse.sourcecred.io/t/grain-a-project-specific-cred-powered-token/147)
+ - [sourcegrain by @decentralion](https://github.com/sourcegrain/mission)
+
+## Community Glossary
+
+The "Community Glossary" is ideal if you want to participate actively
+in the SourceCred community. Terms that are used commonly used in commonity forums,
+issue trackers, or other discussion platforms are included here.
+
+### bikeshedding
+
+#### Definition
+
+Bikeshedding means talking (or even arguing) about a small detail instead of a larger issue at hand. 
+If you imagine building a house and having a bike shed outside, it would be akin
+to angsting over the color and fun name for the bike shed instead of working on
+actually finishing the house. At the end of the day, you won't have a place to live
+because you were lost in details that didn't matter.
+
+#### In the wild
+
+If you take a look at [sourcecred](https://github.com/sourcecred/sourcecred/search?q=bikeshedding&type=Issues), there are several instances of a mention of bikeshedding.
+Here is an example from @decentralion:
+
+> It would probably be best to split this into two PRs, one which adds the badges and one which adds the colors, as color choices are always an ample source of bikeshedding :)
+
+In the above statement, he recognizes that contributors are likely to get distracted
```

Comment:
nit: i use they/them pronouns

Reply:
So sorry about that! I definitely know this, my fingers just subconsciously typed it. I'll try harder in the future to be careful.